src/yunohost/tools.py

```diff
@@ -555,3 +555,63 @@ def tools_diagnosis(auth, private=False):
         diagnosis['private']['domains'] = domain_list(auth)['domains']
 
     return diagnosis
+
+
+def tools_urlavailable(auth, url):
+    """
+    Check availability of a web path
+
+    Keyword argument:
+        url -- Url to check
+    """
+
+    domain, path = _parse_app_url(url)
+
+    # Abort if domain is unknown
+    if domain not in domain_list(auth)['domains']:
+        raise MoulinetteError(errno.EINVAL, m18n.n('domain_unknown'))
+
+    # Fetch apps map
+    apps_map = app_map(raw=True)
+
+    # Loop through all apps to check if path is taken by one of them
+    available = "Yes"
```

Comment:
Shouldn't we uses a boolean value? This is json serializable and will be easier to uses in the admin interface.